src/components/product/Product.js

```diff
@@ -0,0 +1,14 @@
+function Product({labelText, productImage, bagName, bagPrice}) {
+    return (<>
+        <article>
+            <span>{labelText}</span>
+            <img src={productImage} alt=""/>
```

Comment:
Hier zou nog een alt image property bij kunnen. Hierdoor kunnen mensen die gebruik maken van een screenreader "zien" wat de image is.